src/components/routing/Navigation/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+
+function CustomLink({ children, to }) {
```

Comment:
Woooow! 😮 😮 😮  

`CustomLink` component is so neat, and clever solution. Congrats, incredible work! 😄 🥳